src/wallet/wallet.cpp

```diff
@@ -3298,7 +3298,7 @@ bool CWallet::InitLoadWallet()
             CKey key;
             key.MakeNewKey(true);
             if (!walletInstance->SetHDMasterKey(key))
-                throw std::runtime_error("CWallet::GenerateNewKey(): Storing master key failed");
+                throw std::runtime_error("CWallet::InitLoadWallet(): Storing master key failed");
```

Comment:
It is not clear to me when to use `return false` over `throw`.

Wouldn't it make sense to modify SetHDMasterKey() to `return false`?